static/_mitx.scss

```diff
@@ -255,3 +255,39 @@ height: 100px;
 margin-top: 132px;
 background-position: center top !important;
 }
+
+//--------------
+// Jolyon's PRs
+//--------------
+
+@media print {
+    // PR https://github.com/edx/edx-platform/pull/8972
+    .xblock-student_view-discussion, .wrapper-preview-menu {
+        display: none;
+    }
+
+    div.problem .action {
+        .save, .check, .show, .reset, .hint-button {
+            display: none;
+        }
+    }
+
+    div.course-wrapper section.course-content {
+        padding: 4mm 2mm 0;
+    }
+
+    // PR https://github.com/edx/edx-platform/pull/8970
+    .wrapper-footer {
+        display: none;
+    }
+}
+
+// PR https://github.com/edx/edx-platform/pull/7594
+div.problem .inline + p {
```

Comment:
These last two are the only ones I'm a little concerned with, can you split this into two commits, (one with print, one with these), so we can revert one if issues are raised? Other than that little house keeping thing, happy to merge.